Release notes for OS 2.3.1 Patch Release (#322)
farm-ng/amiga-dev-kit: website/docs/release-notes/2024-07-17.md

  - The Brain now accurately tracks the correct date and time across power-offs,
  even when it does not have access to Wi-Fi for NTP updates.

## New Features

- **Recorder Profiles**
  - Users can now create and edit profiles for recording data streams from the robot.
  This feature is particularly useful for customized data collection needs, allowing for
  greater flexibility in recording configurations.
  Check out more in [Launcher - Recorder](/docs/apps/launcher/#recorder)

  ![recorder](https://github.com/user-attachments/assets/68cabb56-1b16-4c45-a047-202910879873)

- **CAN Status and Error States**
  - The app bar now displays CAN status and error states, providing users with real-time
  diagnostics and improving overall system transparency and troubleshooting capabilities.

  ![can-status-app-bar](https://github.com/user-attachments/assets/8364b94b-46ab-4e6b-89d9-e222af7d8e0a)

## Breaking Changes

- **Timestamps**
  - Some events have additional timestamps. Note which one to use for backwards compatibility.
  While it is possible to retrieve event stamps by indexing them by order, we strongly recommend
  you to do so by `semantics` and `clock type`.

>   For context, the available semantics are available on [farm-ng-core: stamp.py](https://github.com/farm-ng/farm-ng-core/blob/main/py/farm_ng/core/stamp.py).

Reviewer: was just a lot of "available"
```suggestion
  For context, the available semantics can be found in [farm-ng-core: stamp.py](https://github.com/farm-ng/farm-ng-core/blob/main/py/farm_ng/core/stamp.py).
```
